hooks/post_gen_project.py

```diff
@@ -1,9 +1,216 @@
+                "Require a designated code owner to"
+                + " approve pull requests (y/n)? [n]: ",
+            "enable_issues":
+                "Enable the GitHub Issues features"
+                + " on the repository (y/n)? [n]: ",
+            "enable_merge_commits": "Allow merge commits (y/n)? [n]: ",
+            "enable_squash_merging": "Allow squash merging (y/n)? [n]: ",
+            "enable_rebase_merging": "Allow rebase merging (y/n)? [n]: ",
+        }
+
+        # Multiple entry questions
+        DICT_QUESTIONS = {
+            "topics": {
+                "start_question": "Add a topic to the repository (y/n)? [n]: ",
+                "init_loop": "What topic would you like to add? "
+            },
+            "required_status_checks": {
+                "start_question":
+                    "Add required checks for merging into"
+                    + " the branch (y/n)? [n]: ",
+                "init_loop":
+                    "What check would you like to add? "
+            },
+            "restricted_pr_teams": {
+                "start_question":
+                    "Restrict which teams can dismiss push requests"
+                    + " (y/n)? [n]: ",
+                "init_loop":
+                    "What team would you like to restrict? "
+            },
+            "restricted_push_teams": {
+                "start_question":
+                    "Restrict who can push to the branch (y/n)? [n]: ",
+                "init_loop":
+                    "What team would you like to restrict? "
+            }
+        }
+
+        directory = "terraform/"
+        os.mkdir(directory)
+        token_q = (
+            "Enter the personal access token for "
+            "creating the organizational repo: ")
+        os.environ['GITHUB_ORGANIZATION'] = (
+            "{{ cookiecutter.github_username }}")
+        os.environ['GITHUB_TOKEN'] = input(token_q).lower()
+
+        terraform_vars = {
+            "name": "{{ cookiecutter.module_name }}",
+            "description": "{{ cookiecutter.short_description }}"
+        }
+
+        for var, question in YN_QUESTIONS.items():
+            choice = check_choice(question)
+            terraform_vars[var] = str(choice).lower()
+
+        for var, question in DICT_QUESTIONS.items():
+            choice = check_choice(question['start_question'])
+            if choice:
+                selection = multiple_choices(question['init_loop'])
+                terraform_vars[var] = selection
+
+        collaborators = get_collaborators()
+        if collaborators:
+            terraform_vars['access_teams'] = collaborators
+
+        with open(directory + "terraform.tfvars.json", "w") as fh_:
+            fh_.write(json.dumps(terraform_vars))
+
+        success = run_terraform(directory, TF_MODULE_URL)
+
+        print("Return Code: {}".format(success))
+        if success == 0:
+            shutil.rmtree(os.path.join(os.getcwd(), directory))
```

Comment:
```suggestion
            shutil.rmtree(join(os.getcwd(), directory))
```

Reply:
realized I wasn't being super consistent with `os.path.join()` vs `join()` throughout the rest of this project so I removed the `from os.path import join` import line and opted for `os.path.join()`

Reply:
Even better :)